src/test/kotlin/com/fasterxml/jackson/module/kotlin/test/failing/Github271AlphaSortProperties.kt

```diff
@@ -16,6 +17,11 @@ class TestGithub271 {
         val mapper = jacksonObjectMapper()
 
         val json = mapper.writeValueAsString(Foo("a", "c"))
-        assertEquals("""{"a":"a","b":"b","c":"c"}""", json)
+        try {
+            assertEquals("""{"a":"a","b":"b","c":"c"}""", json)
+            fail("GitHub #271 has been fixed!")
+        } catch (e: AssertionError) {
+            // Remove this try/catch and the `fail()` call above when this issue is fixed
+        }
```

Comment:
Some tests run fine but have failing assertions, in which case a similar pattern is used but `AssertionError` is caught instead.